ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/db/database_binding.clj

```diff
@@ -52,56 +52,62 @@
       :data
       deserialize))
 
-(defn dispatch-fn   
+(defn dispatch-fn
   [collection-id options]
   collection-id)
 
 (defmulti  find-resources dispatch-fn)
 (defmethod find-resources :default
-  [collection-id options]  
-  (->>  (select resources (where {:id [like (str collection-id"%")]}))
-        (map :data)
-        (map deserialize)))
-
-(defn- delete-resource 
+  [collection-id options]
+  (let [{:keys [offset limit]} (u/offset-limit options)]
+    (if (neg? limit)
+      []
+      (->> (select resources
+                   (where {:id [like (str collection-id"%")]})
+                   (kc/limit offset)
+                   (kc/offset limit))
```

Comment:
Well spotted @rbf 
These 2 lines are actually wrong. The thing is that the new pagination (CIMI like with $first and $last) was well tested end-to-end for the other resource using (`usage`), but not for `event`. 
The highlighted lines concern the general implementation of `find-resources` that apply for `event` resource.